src/Module/Post/Tag/Remove.php

```diff
@@ -78,7 +78,7 @@ protected function content(array $request = []): string
 		$tag_text = Tag::getCSVByURIId($item['uri-id']);
 
 		$tags = explode(',', $tag_text);
-		if (empty($tags)) {
+		if (!is_array($tags)) {
 			$this->baseUrl->redirect($returnUrl);
 		}
```

Comment:
`is_array($tags)` will always be true as the return value of `explode()` won't be `false` (we're setting the delimiter manually. Instead we should check whether `$tag_text` is empty and then do the `explode()` call.
```suggestion
		if ($tag_text === '') {
			$this->baseUrl->redirect($returnUrl);
		}
		
		$tags = explode(',', $tag_text);
```